examples/common_vmseries_and_autoscale/README.md

```diff
@@ -12,7 +12,7 @@ Virtual Machine Scale Sets (VMSS) are used for autoscaling to run the Next Gener
 
 ## Reference Architecture Design
 
-![simple](https://github.com/PaloAltoNetworks/terraform-azurerm-vmseries-modules/assets/6574404/a7c2452d-f926-49da-bf21-9d840282a0a2)
+![simple](https://github.com/PaloAltoNetworks/terraform-azurerm-swfw-modules/assets/6574404/a7c2452d-f926-49da-bf21-9d840282a0a2)
```

Comment:
these images will need to be re-uploaded in order to create github.com assets links. needed for all images.